Libraries/StableDiffusion/Tokenizer.swift

```diff
@@ -36,7 +36,7 @@ struct Bigram: Hashable {
 class CLIPTokenizer {
 
     let pattern =
-        /<\|startoftext\|>|<\|endoftext\|>|'s|'t|'re|'ve|'m|'ll|'d|[\p{L}]+|[\p{N}]|[^\s\p{L}\p{N}]+/
+        #/<\|startoftext\|>|<\|endoftext\|>|'s|'t|'re|'ve|'m|'ll|'d|[\p{L}]+|[\p{N}]|[^\s\p{L}\p{N}]+/#
```

Comment:
Convincing swiftpm to compile this with the bare `/` proved to be impossible (or at least beyond what the docs said should work).